setup.py

```diff
@@ -93,9 +102,12 @@ def run(self):
         _datafiles()
         _develop.run(self)
 
+get_patches()
```

Comment:
That was the first thing I tried, but `_datafiles()` is only being called during the `build` process after packaging. So the `patches` folder won't go into the tarball.